how-to/README.md

```diff
@@ -0,0 +1,40 @@
+# Skipper Prometheus Metrics Collection
+
+The skipper-ingress pods should be configured to be scraped by Prometheus. This
+can be done by Prometheus service discovery using discovery of 
+[Kubernetes services](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config)
```

Comment:
Maybe also add a link to our example configuration https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/prometheus/configmap.yaml#L69